patches/aesthetics.py

```diff
@@ -261,7 +261,53 @@ def reduceMessageLengths(rom, rnd):
         "Is this a delayed April Fool's Joke?",
         "Dayoman is close to getting 120% in 1:20!",
-        "Play as if your footage will go in a\nSummoning Salt video."
+        "Play as if your footage will go in a\nSummoning Salt video.",
+        "I hope you prepared for our date later.",
+        "Isn't this the game where you date a seagull?",
+        "!tele broken, pls fix",
+        "You look pretty good for a guy who probably drowned.",
+        "I heard Matty's a cheater.",
+        "Remember, we race on Sundays.",
+        "Pink Batman delayed this race for lunch.",
+        "This randomizer was made possible by players like you. \n \n Thank you!",
+        "Now with real fake doors!",
+        "You could be doing something productive right now.",
+        "No eggs were harmed in the making of this game.",
+        "I'm helping the goat, \ncatfishing Mr. Write is kinda the goal.",
+        "There are actually two LADX randomizers.",
+        "Did you actually dump this cart?"
+        "You're not gonna cheat... \n ...right?",
+        "Mamu's singing is so bad it wakes the dead.",
+        "Don't forget the Richard picture.",
+        "Are you sure you wanna do this?  I kinda like this island.",
+        "SJ, BT, WW, OoB, HIJKLMNOP.",
+        "$5 in the swear jar.  Now.",
+        "#####, I promise this seed will be better than the last one.",
+        "Want your name here?  Contribute to LADXR!",
+        "Kappa",
+        "Dragon already finished this seed, about an hour ago.",
+        "HEY! \n \n LANGUAGE!",
+        "I sell seashells on the seashore.",
+        "Hey!  Are you even listening to me?",
+        "Your stay will total 10,000 rupees.  I hope you have good insurance.",
+        "I have like the biggest crush on you.  Will you get the hints now?",
+        "Daid watches Matty for ideas. \nBlame her if things go wrong.",
+        "'All of you are to blame.' -Daid",
+        "Batman Contingency Plan: Link.  Step 1: Disguise yourself as a maiden to attract the young hero.",
+        "I have flooded the Wind Fish's Egg with a deadly neurotoxin.",
+        "Ahh, General #####.",
+        "Finally, Link's Awakening!",
+        "Is the Wind Fish dreaming that he's sleeping in an egg?  Or is he dreaming that he's you?",
+        "Save Koholint.  By destroying it.  Huh?  Don't ask me, I'm just a kid!",
+        "There aren't enough women in this village to sustain a civilization.",
+        "So does this game take place before or after Oracles?",
+        "Have you tried the critically acclaimed MMORPG FINAL FANTASY XIV that has a free trial up to level 60 including the Heavensward expansion?",
+        "The thumbs-up sign had been used by the Galactic Federation for ages. Me, I was known for giving the thumbs-down during briefing. I had my reasons, though... Commander Adam Malkovich was normally cool and not one to joke around, but he would end all of his mission briefings by saying, 'Any objections, Lady?'",

```

Comment:
This might give issues with text table space.

Reply:
I'll give it a shot, see if it runs into any issues.

Reply:
I genned a seed with this text, I didn't run into issues with generation

Reply:
I don't know what kinds of issues to look out for, So I wouldn't know how to bug-hunt on this one, sorry